test/layouts/cartesian_layout.cpp

```diff
@@ -247,3 +247,11 @@ TEST_CASE("Cartesian cardinal operations", "[cartesian-layout]")
                 std::set<cartesian_layout<offset::ucoord_t>::coordinate>{{{4, 5}, {5, 4}, {6, 5}, {5, 6}}}.count(adj));
         });
 }
+
+TEST_CASE("Cartesian layouts with SiQAD coordinates must have a z dimension of 1")
+{
+    using lyt = cartesian_layout<siqad::coord_t>;
+
+    CHECK(lyt{aspect_ratio<lyt>{0, 0}}.z() == 1);
+    CHECK(lyt{aspect_ratio<lyt>{9, 9}}.z() == 1);
```

Comment:
Just for completeness, also add one check that initializes the layout with an aspect ratio that already has a z value of 1.